_pydevd_sys_monitoring/_pydevd_sys_monitoring.py

```diff
@@ -322,7 +321,7 @@ def _create_thread_info(depth):
     if t is None:
         t = _thread_active.get(thread_ident)
 
-    if isinstance(t, threading._DummyThread):
+    if isinstance(t, threading._DummyThread) and not IS_PY313_OR_GREATER:
```

Comment:
This was no longer necessary in 3.13. Or at least it caused crashes in tests.

Reply:
Yes, I provided a PR to CPython itself that does this now, so, special handling should no longer be needed as it was added to 3.13.